apps/README.md

```diff
@@ -189,6 +189,52 @@ fetching files served by the test runner, prefix the file path with
 document.write('<audio src="/base/test/audio/assets/win.mp3"/>');
 ```
 
+#### UI Component Style Guide ####
+
+We use `react-storybook` to generate a ui component style guide that you can use
+to discover what components are available to reuse as you build new
+features. You can also use the style guide to more easily develop new components
+without having to run all of code.org.
```

Comment:
OK, to answer one of my questions, I see below that I develop a component and then follow the steps below to add it to the storybook. Another question: should components which are deeply nested within an app be added to the storybook? or, asked differently, if I'm going to add a component to the storybook, should I extract it out to some common place first outside of e.g. `src/applab` first?